package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cros-dpsl-js",
-  "version": "1.0.2",
+  "version": "1.2.0",
```

Comment:
This PR must be merged after #13

Reply:
What is the strategy of updating version number, e.g. why it's next version not 1.0.3?

WDYT about syncing DPSL version with Chrome OS version where all APIs expected to work? This will help with version control.

Reply:
The reason is to follow npm and Javascript ecosytem standards by using semantic versioning. https://docs.npmjs.com/about-semantic-versioning

Since this PR is about adding new feature (new API functions), then it is a Minor release.

Reply:
ack. Thanks for more info!